.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
+- [ ] Install the latest version of mlr: `update.packages(oldPkgs="mlr", ask=FALSE)`
+- [ ] If you use a GitHub install of mlr: `devtools::install_github(c("BBmisc", "ParamHelpers", "mlr"))
```

Comment:
So how is someone supposed to tick all these boxes? Install the latest cran `mlr` version and *then* install it from github again?